Source/Core/Core/ConfigManager.cpp

```diff
@@ -1055,28 +1077,78 @@ IniFile SConfig::LoadGameIni() const
   return LoadGameIni(GetGameID(), m_revision);
 }
 
+void SConfig::MergeDefaultGameIni(const std::string& id, std::optional<u16> revision, IniFile* out)
+{
+  for (const std::string& filename : ConfigLoaders::GetGameIniFilenames(id, revision))
+  {
+    out->Load(File::GetSysDirectory() + GAMESETTINGS_DIR DIR_SEP + filename, true);
```

Comment:
Why does this function try to load from both the old location and from the zip? As far as I can tell, either we're not going to have any loose INIs and this line of code will just waste time, or we are going to have old loose INIs leftover (either due to the (Windows/Android only?) buildbot not being cleaned out or due to the user extracting the new Dolphin .7z file on top of an old install) and we will read stale game settings.